MBW.py

```diff
@@ -68,34 +61,29 @@ def fadeOut(player, seconds):
 
 
 def test(player):
-  elapsed = 0
+  elapsed = 0 # not used, for timeout on test function?
   invasions = 0
   invading = 0
   while True:
     # TODO loop chill bpm
     if not int(time()) % 5:
 
-      if signaler.invader_detect() and invading <= 0:
-        player.set_media(instance.media_new(choice(cleaned_list)[0]))
-        fadeIn(player, 2)
-        print('New invasion # {}'.format(invasions))
-        invasions += 1
-        invading = 2
-
-      elif signaler.invader_detect() and invading > 0:
-        invading = 2
-
+      if signaler.invader_detect():
+        if invading <= 0:
+          player.set_media(instance.media_new(choice(cleaned_list)[0]))
+          fadeIn(player, 2)
+          print('New invasion # {}'.format(invasions))
+          invasions += 1
+        else:
+          invading = 2
+      else:
+        invading -= 1
```

Comment:
At a glance I think this will work. I will test this tomorrow.

Reply:
Okay, let me know how it goes.

Reply:
Cool. Let me know.